ios/tools/prepRelease.sh

```diff
@@ -56,27 +56,56 @@ LC_ALL=UTF-8 sed -e "s/[$(printf $emoji)]//g" < "$CHANGELOG_PATH" > "$CHANGELOG_
 mv -f "$CHANGELOG_PATH.1" "$CHANGELOG_PATH"
 assertFileExists "${CHANGELOG_PATH}"
 
+KMEI_DST_NAME="keyman-engine-ios-${BUILD_NUMBER}.zip"
+KMEI_DST="${WORK_DIR}/${UPLOAD_DIR}/${KMEI_DST_NAME}"
+
 #
-# Keyman Engine
+# Copy xcframeworks from Carthage to upload directory and add to zip
 #
 
-KMEI_DST_NAME="keyman-engine-ios-${BUILD_NUMBER}.zip"
-KMEI_DST="${WORK_DIR}/${UPLOAD_DIR}/${KMEI_DST_NAME}"
+CARTHAGE_DIR="Carthage"
+KEYMAN_FRAMEWORKS="Carthage/Build/"
+#KEYMAN_FRAMEWORK_BASE="Carthage"
 
-KMEI_FRAMEWORK_BASE="build/Build/Products/Release/"
-FRAMEWORK="KeymanEngine.xcframework"
+cd $UPLOAD_DIR
```

Comment:
```suggestion
cd "$UPLOAD_DIR"
```

Here and elsewhere, always quote variables to avoid space tragedies